app/views/articles/_object.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="list-group-item">
-  <%= tag.span class: "text-lg", link_to(object.title.t,
-                  object_path(object, q: get_query_param)) %>
+  <%= tag.span link_to(object.title.t, object_path(object, q: get_query_param)),
```

Comment:
`tag` parameters are partly reversed from `content_tag`. In particular content comes before options.
See https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag